File: sangi_practice/sangi_practice_05/model_answers/cpp/main.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    int cnt = 0;
    while (n > 0) {
        cnt += n & 1;
        n >>= 1;
    }
    cout << cnt << "\n";
    return 0;
}

File: sangi_practice/sangi_practice_05/model_answers/java/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int cnt = 0;
        while (n > 0) {
            cnt += n & 1;
            n >>= 1;
        }
        System.out.println(cnt);
    }
}
